docs/root/configuration/http_conn_man/headers.rst

```diff
@@ -118,15 +118,21 @@ should be replaced by backslash-double-quote (\").
 
 The following keys are supported:
 
-1. ``By`` The Subject Alternative Name (SAN) of the current proxy's certificate.
+1. ``By`` The Subject Alternative Name (URI type SAN) of the current proxy's certificate.
 2. ``Hash`` The SHA 256 diguest of the current client certificate.
-3. ``SAN`` The SAN field (URI type) of the current client certificate.
+3. ``SAN`` The URI type SAN field of the current client certificate.
 4. ``Subject`` The Subject field of the current client certificate. The value is always double-quoted.
+5. ``DNS`` The DNS type SAN field of the current client certificate. A client certificate may contain multiple DNS Subject Alternative Names, each will be a separate entry in XFCC header.
```

Comment:
While you're here, could you also document `Cert`? It looks that it was missed in commit that added it.

Reply:
Sure, updating.